experimental/ag-ui/server-agui.py

```
@@ -227,13 +534,36 @@ async def event_generator(message_history):
+                model=model_name,
+                agent_name="HolmesGPT",
+                success=False,
+            )
+
+            logging.info(
+                f"[AG-UI] Sending RUN_ERROR for run_id={input_data.run_id}"
+            )
             yield encoder.encode(
                 RunErrorEvent(
                     type=EventType.RUN_ERROR,
                     message=f"Agent encountered an error: {str(e)}",
```

Ruff (0.14.11)

Use explicit conversion flag

Replace with conversion flag

(RUF010)